cfgov/cfgov/urls.py

```diff
@@ -371,25 +371,25 @@ def empty_200_response(request, *args, **kwargs):
     re_path(r'^es/obtener-respuestas/([-\w]{1,244})-(es)-(\d{1,6})/imprimir/$',
         view_answer,
         name='ask-spanish-answer'),
-    re_path(r'^(?i)ask-cfpb/search/$',
+    re_path(r'^ask-cfpb/search/$(?i)',
         ask_search,
         name='ask-search-en'),
-    re_path(r'^(?i)ask-cfpb/search/(?P<as_json>json)/$',
+    re_path(r'^ask-cfpb/search/(?P<as_json>json)/$(?i)',
         ask_search,
         name='ask-search-en-json'),
-    re_path(r'^(?i)ask-cfpb/api/autocomplete/$',
+    re_path(r'^ask-cfpb/api/autocomplete/$(?i)',
         ask_autocomplete, name='ask-autocomplete-en'),
     re_path(r'^(?P<language>es)/obtener-respuestas/api/autocomplete/$',
         ask_autocomplete, name='ask-autocomplete-es'),
 
     re_path(r'^_status/', include_if_app_enabled('watchman', 'watchman.urls')),
 
     re_path(
-        r'^(?i)consumer-tools/financial-well-being/',
+        r'^consumer-tools/financial-well-being/',
         include('wellbeing.urls')
     ),
     re_path(
-        r'^(?i)about-us/diversity-and-inclusion/',
+        r'^about-us/diversity-and-inclusion/',
         include((
             'diversity_inclusion.urls',
             'diversity_inclusion'),
```

Comment:
We should ensure that `diversity_inclusion/urls.py` has similar changes so that those URLs remain case-insensitive.

Reply:
Also addressed in PR https://github.com/cfpb/cfgov-refresh/pull/5649